test/test-rotateElement.js

```diff
@@ -0,0 +1,203 @@
+var assert = require('chai').assert;
+var tetris = require('../tetris')
+describe("element rotation around the axis 3x3", function () {
+
+    it("element move the top", function () {
+        let elementTetromino = tetris.getCoordinates(`
+            ---
+            --X
+            ---
+        `)
+
+        let fulcrum = tetris.getCoordinates(`
+            ---
+            -X-
+            ---
+        `)
+        let stepElementTetromino = tetris.rotateElement(fulcrum, elementTetromino)
+        let expectedResult = tetris.getCoordinates(`
+			-X-
+			---
+			---
+        `)
+        assert.deepEqual(expectedResult, stepElementTetromino)
+    })
+
+    it("element move the left", function () {
+        let elementTetromino = tetris.getCoordinates(`
+            ---
+            --X
```

Comment:
Incorrect
Exactly the same input as in previous test. Please check all your tests

Reply:
I thought it should be so.
I will turn the same coordinate.

Reply:
No
All tests should have different inputs and different outputs 
Currently, you have same inputs and different output, which is impossible
Please change inputs to all your tests

Reply:
Done